packages/@react-spectrum/s2/package.json

```diff
@@ -122,7 +122,11 @@
   "devDependencies": {
     "@adobe/spectrum-tokens": "^13.0.0-beta.53",
     "@parcel/macros": "^2.13.0",
-    "@react-aria/test-utils": "1.0.0-alpha.1"
```

Comment:
not sure how this one fell behind, but I've updated it to the latest local version so we don't have two copies installed